report_py3o/models/py3o_template.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 XCG Consulting (http://odoo.consulting)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import fields, models
+
+
+class Py3oTemplate(models.Model):
```

Comment:
IMO we're switching naming schemas all over the place, and not many people will know what py3o is. Please switch this to ODFTemplate or similar